jsnlog.ts

```diff
@@ -767,14 +767,8 @@ module JL
                 // If it doesn't get re-enabled, amount of cpu cycles wasted is minimal.
                 if (!allow(this)) { return; }
 
-                // If a request is in progress, abort it.
-                // Otherwise, it may call the success callback, which will be very confusing.
-                // It may also stop the inflight request from resulting in a log at the server.
-
-                var xhrState = this.xhr.readyState;
-                if ((xhrState != 0) && (xhrState != 4)) {
-                    this.xhr.abort();
-                }
+                // Because a react-native XMLHttpRequest cannot be reused it needs to be recreated with each request
+                this.xhr = JL._createXMLHttpRequest();
```

Comment:
I appreciate every request has to be sent with a new XMLHttpRequest object.

However, you are also removing the code that detects whether the previous request is still in progress, and aborts it if it is still in progress.

I would have thought that it would make sense to keep that functionality:
1) check if this.xhr is defined. If so, then if readyState is not 0 or 4 (meaning previous request in progress), abort that request 
2) create new XMLHttpRequest and assign to this.xhr

Can you see any drawbacks to keeping the abort functionality?
If not, please modify your pull request.

Reply:
@mperdeck you are right, i've updated the PR with this change